js/fizzbuzz.js

```diff
@@ -0,0 +1,67 @@
+'use strict'
+
+var FizzBuzz = (function() {
+
+  var array = [];
+  var fizzBuzz = function (firstWord, secondWord) {
+
+    if (firstWord !== '') {
+      this.firstWord = firstWord;
+    }
+    else {
+      this.firstWord = 'Fizz';
+    }
+
+    if (secondWord !== '') {
+      this.secondWord = secondWord;
+    }
+    else {
+      this.secondWord = 'Buzz';
+    }
+
+    console.log(this.firstWord);
+
+  }
+
+  fizzBuzz.prototype.readArray = function(lowNumber, highNumber) {
+
+    for (var i = 0; i < (highNumber - lowNumber + 1); i++) {
+
+      array[i] = lowNumber + i;
```

Comment:
Because of how you're assigning these values, you actually end up in a scenario where a user gets an unexpected result. Try this out in your index.html page:

1. Enter `10` for your low number
1. Enter `3000` for your high number
1. Submit the form
1. Look at the output - it starts at 10 and goes all the way to 3000
1. Enter 1 for your low number
1. Enter 100 for your high number
1. Look at the updated output - it starts at 1, but goes to 3000 still.

What's happening here is that you're adding the values for 1-9, overwriting the values of 10-100, and then you still have the values for 101-3000 in your array.

If, in fact, you want your users to always see the highest values they've entered, then great! But I suspect that wasn't what you'd intended. You have a couple of options here -- either reset the value of the global `array` variable at the beginning of this function (`array = [];`) OR use a variable that is scoped to `readArray`:

```javascript
fizzBuzz.prototype.readyArray = function(lowNumber, highNumber) {
  var array = [];
  /* process everything */
  return array;
}
```

_Finally_ - I'd recommend against using `array` as a variable name; it's both too generic (can literally be anything) and also potentially ends up colliding with the reserved keyword of `Array` (they _are_ different, since JS is case-sensitive, but it's a best practice to steer clear of those keywords in any format). Here I would have probably gone with either `values` or `output`.